vagrant/boot_cluster.sh

```diff
@@ -4,8 +4,8 @@ set -ex
 
 # Launch and wait for toxiproxy
 ${REPOSITORY_ROOT}/vagrant/run_toxiproxy.sh &
-while ! nc -q 1 localhost 2181 </dev/null; do echo "Waiting"; sleep 1; done
-while ! nc -q 1 localhost 9092 </dev/null; do echo "Waiting"; sleep 1; done
+while ! nc -q 1 localhost 2185 </dev/null; do echo "Waiting"; sleep 1; done
+while ! nc -q 1 localhost 9095 </dev/null; do echo "Waiting"; sleep 1; done
```

Comment:
Were these ports change intended ?

Reply:
Yeah these `nc` polls are waiting for toxiproxy to have completed its startup. However, it starts 5 zookeeper listeners and 5 kafka listeners so it makes sense to poll the highest port numbers (which are created last) for zk-5 and kafka-5 respectively